index.js

```diff
@@ -47,6 +52,8 @@ module.exports = {
         projectName: this.parent.pkg.name,
         projectTag: this.parent.pkg.version,
         projectHref: info && info.browse(),
+        primaryBranch: userConfig.getPrimaryBranch(),
+        latestVersionName: LATEST_VERSION_NAME,
```

Comment:
Should this be `latestVersionKey`/`LATEST_VERSION_KEY` now?

Reply:
Yes, probably worth renaming.